games/fight_big_parma.js

```diff
@@ -0,0 +1,637 @@
+/*
+@title: fight_big_parma
+@author: me, myself, and that guy
```

Comment:
`@author: me, myself, and that guy`
haha, are you sure?

Reply:
quite

Reply:
Is that a "are you really sure" or "please change this"

Reply:
"jakempock" isn't my real name or primary github anyway, so it doesn't really matter to me

Reply:
It's because it's not your nickname or your name

Reply:
alright, changing now